examples/mnist_ddp.py

```diff
@@ -141,14 +142,14 @@ def main():
     )
     parser.add_argument('--save-model', action='store_true', default=False, help='For Saving the current Model')
 
-    parser.add_argument('--local-rank', type=int, help='local rank, will passed by ddp')
-
     parser.add_argument('--enable-msamp', action='store_true', default=False, help='enable MS-AMP')
     parser.add_argument('--opt-level', type=str, default='O1', help='MS-AMP optimization level')
 
     args = parser.parse_args()
     use_cuda = not args.no_cuda and torch.cuda.is_available()
 
+    args.local_rank = int(os.environ.get('LOCAL_RANK', '0'))
```

Comment:
It will be more compatible.
```python
    parser.add_argument('--local-rank', type=int, help='local rank, will passed by ddp')
    if args.local_rank is None and 'LOCAL_RANK' in os.environ:
        args.local_rank = int(os.environ['LOCAL_RANK'])
```

Reply:
Sure. Fixed.